packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -401,6 +407,23 @@ public void updateClippingRect() {
     updateClippingToRect(mClippingRect);
   }
 
+    @Override
+  public void startViewTransition(View view) {
+    super.startViewTransition(view);
+    mIsTransitioning = true;
+  }
+
+  @Override
+  public void endViewTransition(View view) {
+    super.endViewTransition(view);
+    mIsTransitioning = false;
+    if (mChildrenRemovedWhileTransitioning != null) {
+      mChildrenRemovedWhileTransitioning.clear();
```

Comment:
I'm not sure whether `HashMap.clear` deallocates or not. If not, this call is redundant.

Reply:
Outdated. We don't want to null the reference here.